apps/openchallenges/mcp-server/src/main/java/org/sagebionetworks/openchallenges/mcp/server/ChallengeService.java

```diff
@@ -0,0 +1,112 @@
+package org.sagebionetworks.openchallenges.mcp.server;
+
+import io.micrometer.common.lang.Nullable;
+import java.time.LocalDate;
+import java.util.List;
+import org.sagebionetworks.openchallenges.api.client.api.ChallengeApi;
+import org.sagebionetworks.openchallenges.api.client.model.ChallengeCategory;
+import org.sagebionetworks.openchallenges.api.client.model.ChallengeDirection;
+import org.sagebionetworks.openchallenges.api.client.model.ChallengeIncentive;
+import org.sagebionetworks.openchallenges.api.client.model.ChallengeSearchQuery;
+import org.sagebionetworks.openchallenges.api.client.model.ChallengeSort;
+import org.sagebionetworks.openchallenges.api.client.model.ChallengeStatus;
+import org.sagebionetworks.openchallenges.api.client.model.ChallengeSubmissionType;
+import org.sagebionetworks.openchallenges.api.client.model.ChallengesPage;
+import org.springframework.ai.tool.annotation.Tool;
+import org.springframework.ai.tool.annotation.ToolParam;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ChallengeService {
+
+  private final ChallengeApi challengeApi;
+
+  public ChallengeService(ChallengeApi challengeApi) {
+    this.challengeApi = challengeApi;
+  }
+
+  @Tool(
+    name = "list_challenges",
+    description = """
+    Lists challenges that can be filtered and sorted based on a variety of criteria.
+
+    Guidelines for using this tool:
+    - If a parameter is not specified, the default value should be used.
+    - If the user does not specify a challenge status, do not apply any status filter - search across all challenges status by default.
+    - If the user mentions "Docker" or "Container", set: submissionTypes = ["container_image"]
```

Comment:
Be more specific: submission types should include a specific value but it should also accept other values.